### PR TITLE
fix: render_to_response is not defined

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: admin_tools/menu/views.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def remove_bookmark(request, id):
                 "title": "**title**",  # replaced on the javascript side
             },
         )
-    return render_to_response(
+    return render(
```
Comment (ddimmich) [inline admin_tools/menu/views.py]: Hi - don't you also want to remove render_to_response on the import line?

**Comment by ddimmich** [comment]
This works for me - any chance of this being merged and pushed to an updated version?

**Comment by mxschmitt** [comment]
I've installed it directly via my branch:
```
pip install git+https://github.com/mxschmitt/django-admin-tools@patch-1
```

that works.

@izimobil please publish a new version.

**Comment by yakky** [comment]
Thanks @mxschmitt I will merge this with #101 with test

**Comment by yakky** [comment]
Merged with #101

**Pull request closed without merging**